build.gradle

```diff
@@ -103,7 +103,7 @@ testlogger {
 spotbugsMain {
     excludeFilter = file("checkstyle/findbugs-exclude.xml")
     effort = 'max'
-    ignoreFailures = false
+    ignoreFailures = true // TODO: Set this to false later as they are too many warnings to be fixed.
```

Comment:
If we are not planning to fix this now, can we open an issue and link it here?

Looks like you fixed almost all the issues.

Reply:
I have created issue - https://github.com/opensearch-project/performance-analyzer-rca/issues/157 to track this.

There are still many medium related warnings which needs to be fixed. I fixed high priority ones.